<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBreakpointDialog</class>
 <widget class="QDialog" name="EditBreakpointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Breakpoint</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="6" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QCheckBox" name="enabledCheckBox">
     <property name="toolTip">
      <string>Select, whether the breakpoint is enabled</string>
     </property>
     <property name="text">
      <string>Enabled</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QCheckBox" name="temporaryCheckBox">
     <property name="toolTip">
      <string>Select whether this is a temporary breakpoint</string>
     </property>
     <property name="text">
      <string>Temporary Breakpoint</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="filenameCombo">
     <property name="toolTip">
      <string>Enter the filename of the breakpoint</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QComboBox" name="conditionCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Enter or select a condition for the breakpoint</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="ignoreSpinBox">
     <property name="toolTip">
      <string>Enter an ignore count for the breakpoint</string>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="linenoSpinBox">
     <property name="toolTip">
      <string>Enter the linenumber of the breakpoint</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2_2">
     <property name="text">
      <string>Linenumber:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>Filename:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Condition:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Ignore Count:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="fileButton">
     <property name="toolTip">
      <string>Press to open a file selection dialog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>filenameCombo</tabstop>
  <tabstop>linenoSpinBox</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>conditionCombo</tabstop>
  <tabstop>ignoreSpinBox</tabstop>
  <tabstop>temporaryCheckBox</tabstop>
  <tabstop>enabledCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditBreakpointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditBreakpointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
